### PR TITLE
New Rules Published

### PR DESCRIPTION
**Pull request by faisalusuf**
New detection rules are added to the rules/cloud/github repo.

To configure the log source use the audit log streaming feature of GitHub enterprise. The logs can be ingested in any of the SIEM solutions.

Detailed Guide can be found [here](https://docs.github.com/en/enterprise-cloud@latest/admin/monitoring-activity-in-your-enterprise/reviewing-audit-logs-for-your-enterprise/streaming-the-audit-log-for-your-enterprise#setting-up-audit-log-streaming)

### CODE REVIEW

File: rules/cloud/github/dependabot_alerts_disabled.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+title: Dependabot Alerts Disabled
```
Comment (frack113) [inline rules/cloud/github/dependabot_alerts_disabled.yml:1]:
```suggestion
title: Disabled Outdated Dependency or Vulnerability Alert
```

File: rules/cloud/github/github_high_risk_configuration_change.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
+title: Github High Risk Configuration Change
```
Comment (frack113) [inline rules/cloud/github/github_high_risk_configuration_change.yml:1]:
```suggestion
title: Disable Github High Risk Configuration
```

File: rules/cloud/github/github_new_org_member_alert.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+title: Github New Org Member Alert
```
Comment (frack113) [inline rules/cloud/github/github_new_org_member_alert.yml:1]:
```suggestion
title: Github New Org Member
```

**Review by nasbench — APPROVED** [review]

**Comment by nasbench** [comment]
Hi,

I've updated the metadata of the rules such as including missing tactics of the techniques you included and re-worded some titles. I've also renamed some files to have the same structure as the others. (The rule of thumb is to use a prefix).

I'm not that familiar with GitHub logs but the rules from what I can tell so far LGTM.

Thanks for your contribution once again :)

Regards.

**Comment by faisalusuf** [comment]
Thanks alot for making adjustments. I will review and ensure compliance in future. All the rules being created are well tested in the prod for months.

**Pull request merged**